staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/fuzzer/fuzzer.go

```diff
@@ -66,6 +66,11 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 					{Name: "Age", Type: "date", Description: swaggerMetadataDescriptions["creationTimestamp"], JSONPath: ".metadata.creationTimestamp"},
 				}
 			}
+			if len(obj.SelectableFields) == 0 {
```

Comment:
try `c.Fuzz(obj.SelectableFields`.  without checking for conversions or defaulting, we should be able to fuzz this section.